rfcs/rfc00XX.md

```diff
@@ -0,0 +1,257 @@
+# Built-in lexical exporting
+
+## Preamble
+
+    Author:  Curtis Poe <curtis.poe@gmail.com>
+    Sponsor:
+    ID:      XX
+    Status:  Draft
+
+## Abstract
+
+This RFC proposes adding built-in lexical exporting via an `:export`
+attribute.
+
+## Motivation
+
+To eliminate the need for the myriad different CPAN modules for exporting
+functions. This RFC is only for functions, not variables.
+
+## Rationale
+
+There are many exporter modules available. The core `Exporter` is the best
+known, and has the curious feature of allowing procedural modules to use
+inheritance to implement exporting via package variables. It has a clumsy
+interface and many alternatives arose:
+
+* [Sub::Exporter](https://metacpan.org/pod/Sub::Exporter)
+* [Exporter::Lite](https://metacpan.org/pod/Exporter::Lite)
+* [Exporter::Easy](https://metacpan.org/pod/Exporter::Easy)
+* [Exporter::Tiny](https://metacpan.org/pod/Exporter::Tiny)
+
+And more, including many modules which manually implement exporting directly
+in the `import` method.
+
+There are also many attribute-based exporters:
+
+* [Exporter::Simple](https://metacpan.org/pod/Exporter::Simple)
+* [Perl6::Export](https://metacpan.org/pod/Perl6::Export) (not really attributes)
+* [Perl6::Export::Attrs](https://metacpan.org/pod/Perl6::Export::Attrs)
+
+And so on.
+
+## Specification
+
+Because lower-case attribute names are reserved for Perl's future expansion,
+we do not need a feature guard. Instead, we can simply `use v5.XX` to pull in
+the new feature.
+
+    package My::Module {
```

Comment:
Small note on markdown formatting: I usually like to use the triple-backtick guard for these, so I can add the word "perl" and get better syntax highlight.

    ```perl
    code here ...
    ```